great_expectations/rule_based_profiler/types/data_assistant_result/data_assistant_result.py

```diff
@@ -141,15 +146,14 @@ def get_line_chart(
                         title=domain_title,
                     ),
                     y=alt.Y(metric_name, type=metric_type, title=metric_title),
-                    stroke=point_color_condition,
-                    fill=point_color_condition,
+                    color=point_color_condition,
                     tooltip=tooltip,
                 )
             )
         else:
             points: alt.Chart = (
                 alt.Chart(data=df, title=title)
-                .mark_point(stroke=point_color, fill=point_color, opacity=1.0)
+                .mark_point(opacity=1.0)
```

Comment:
Could this (opacity) be a theme parameter?

Reply:
Where there's a will, there's a way.

Reply:
✅